source/gent/tier1_hortense.rst

```diff
@@ -716,6 +758,46 @@ If users can provide a proper case and motivation, you can contact support to re
 to give you access to all the available resources.
 
 
+Gradual decommissioning
+-----------------------
+
+End 2025, the gradual decommissioning of Tier-1 Compute Hortense will be initiated.
+Around this time, the 4th VSC Tier-1 cluster will become available at the VUB datacenter.
+
+The entire Rome partition is end of life November 2025, and will be shut down by end 2025.
+This implies that the partitions ``cpu_rome``, ``cpu_rome_all``, ``cpu_rome_512``, ``debug_rome`` will all disappear.
```

Comment:
quid the `a100_40`gpu partition? If you are not sure, i would already mention that it might also be removed

Reply:
Very good suggestion.
Statement made